docs/content/en/docs/getting-started/_index.md

```diff
@@ -42,6 +42,25 @@ helm repo update
 
 Alternatively, you may download the individual charts directly from the project's [releases](https://github.com/k8ssandra/k8ssandra/releases) page.
 
+## Enabling Stargate
+
+Stargate deployment is *disabled* by default, to enable it, the values provided to the helm install/upgrade of the
+`k8ssandra-cluster` chart can be modified.
+
+This can be done by adding to the values.yaml file provided to helm:
+
+```
+stargate:
+  enabled: true
+  replicas: 1
```

Comment:
`replicas` is set to 1 by default. I would omit this here; I don't think it's a good idea to encourage redundant configuration.

Reply:
Make sense.  Done.